conf/scaleway.conf

```diff
@@ -0,0 +1,33 @@
+[SERVICE]
+    Flush        5
+    Daemon       off
+    Log_Level    info
+    Parsers_File parsers.conf
+    Log_Level  debug
+    http_debug.request_headers  on
+    http_debug.request_payload  on
+    http_debug.response_headers on
+    http_debug.response_payload on
+
+[INPUT]
+    Name              tail
+    Path              /var/log/*.log
+    Exclude_Path      /var/log/lastlog
+    Path_Key          filename
+    Tag               logs.*
+    DB                /tmp/flb_kube.db
+    Parser            docker
+    Mem_Buf_Limit     5MB
+    Skip_Long_Lines   On
+    Refresh_Interval  10
+
+[OUTPUT]
+    Name              scaleway
+    Match             logs.*
+    Port              443
+    project_id        46fd79d8-1a35-4548-bfb8-03df51a0ebae
```

Comment:
This feels like something that should note be here - maybe an env var would be better.